components/brave_wallet_ui/components/desktop/popup-modals/sell-asset-modal/sell-asset-modal.tsx

```diff
@@ -107,9 +109,23 @@ export const SellAssetModal = (props: Props) => {
     return Number(sellAmount) > new Amount(fiatBalance.format(2)).toNumber()
   }, [sellAmount, fiatBalance])
 
+  const meetsMinimumSellThreshold = React.useMemo(() => {
+    return Number(sellAmount) >= MINIMUM_SELL_THRESHOLD
+  }, [sellAmount])
```

Comment:
Should we move this to the `useMultiChainSellAssets` hook?

Reply:
Kept in `sell-asset-modal.tsx` to reduce complexity.